src/Utils/deleteCommentByID.jsx

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const myApi = axios.create({
+  baseURL: "https://rafaels-nc-news.onrender.com/api",
```

Comment:
Great that you're using axios and the baseURL functionality that comes with creating and instance
You could make better use of this functionality if all your API functions were in one file, because then you would only have to specify the baseURL in one place. This mean that if your API url ever changed, you would only need to change it in one place and removed repetition